nautobot_chatops/utils.py

```diff
@@ -155,11 +212,12 @@ def check_and_enqueue_command(
         return response
 
     # If we made it here, we're permitted. Enqueue it for the worker
-    get_queue("default").enqueue(
-        registry[command]["function"],
+    enqueue_task(
+        command,
         subcommand,
         params=params,
         dispatcher_class=dispatcher_class,
         context=context,
+        function=registry[command]["function"],
```

Comment:
Alternate idea - use keyword-only arguments plus `**kwargs` to handle the difference between required parameters, i.e.:

```python
    enqueue_task(
        function=registry[command]["function"],
        command=command,
        subcommand=subcommand,
        params=params,
        dispatcher_class=dispatcher_class,
        context=context,
    )
```

```python
# Celery
def enqueue_task(*, command, subcommand, params, dispatcher_class, context, **kwargs):
```

```python
# RQ
def enqueue_task(*, function, subcommand, params, dispatcher_class, context, **kwargs):
```

Reply:
This is the one thing I am not good at in Python is kwargs... So, I tend to avoid them.